packages/config/tests/cli/tests.js

```diff
@@ -39,7 +39,7 @@ test('Does not stabilitize output without the --stable flag', async t => {
 
 // This test is too slow in local development
 if (isCI) {
-  test.only('Handles big outputs', async t => {
+  test('Handles big outputs', async t => {
```

Comment:
This was a `test.only()` accidentally left in our code! Ava linting spotted it :)